pkg/rule/rule_manager.go

```diff
@@ -39,43 +37,39 @@ func ManagerInit(wg *sync.WaitGroup, c *nftables.Conn, ruleTarget RuleTarget, f
 }
 
 // Start the rule manager goroutine
-func (r *ManagedRules) Start() {
+func (r *ManagedRules) Start(ctx context.Context) error {
```

Comment:
There isn't a path where we don't return `nil`, but this still seems like the right interface going forward. Maybe there's some setup we do in the future that does fail or w/e.

Reply:
💯 that was my thinking as well.